main/tests/cypress/cypress/integration/project/grid/column/edit-cells/common-transforms/to-text.spec.js

```diff
@@ -0,0 +1,18 @@
+describe(__filename, function () {
+  it('Ensure text is converted to Uppercase', function () {
+    cy.loadAndVisitProject('food.mini');
+
+    cy.columnActionClick('Shrt_Desc', [
```

Comment:
Can we first convert the column to the number and then test the conversion to text?

Reply:
I have changed the code as suggested.